bindings/aws/s3/s3.go

```diff
@@ -44,39 +71,168 @@ func (s *AWSS3) Init(metadata bindings.Metadata) error {
 	if err != nil {
 		return err
 	}
-	uploader, err := s.getClient(m)
+	session, err := s.getSession(m)
```

Comment:
Please, also implement the Close() method.

Reply:
Added, but it's an empty method. I didn't found any method to close AWS session